proto/motor/v1/response.proto

```diff
@@ -4,6 +4,7 @@ syntax = "proto3";
 package sonrio.motor.api.v1;
 
 import "bucket/where_is.proto";
+import "google/protobuf/any.proto";
```

Comment:
Why the unused import?

Reply:
should be removed from a previous implementation